corehq/apps/sms/tasks.py

```diff
@@ -283,7 +283,10 @@ def decrement(self):
 
     @property
     def current_usage(self):
-        return self.client.get(self.key) or 0
+        current_usage = self.client.get(self.key)
```

Comment:
Will all interactions like this with the raw redis client return bytes now that we are on Python 3? Should we be doing something like the final part of this anwer? https://stackoverflow.com/a/44032018/10840